src/pages/workspaces/workspace/DeleteWorkspaceModal.js

```diff
@@ -95,31 +116,33 @@ const DeleteWorkspaceModal = ({ workspace: { workspace: { namespace, name, bucke
     }
   }
 
-  const isDeleteDisabledFromApps = hasApps() && !_.isEmpty(nonDeletableApps)
-
   return h(Modal, {
     title: span({ style: { display: 'flex', alignItems: 'center' } }, [
       icon('warning-standard', { size: 24, color: colors.warning() }),
       span({ style: { marginLeft: '1ch' } }, ['Delete workspace'])
     ]),
     onDismiss,
     okButton: h(ButtonPrimary, {
-      disabled: _.toLower(deleteConfirmation) !== 'delete workspace' || isDeleteDisabledFromApps,
+      disabled: _.toLower(deleteConfirmation) !== 'delete workspace' || isDeleteDisabledFromResources,
       onClick: deleteWorkspace,
       tooltip: Utils.cond(
-        [isDeleteDisabledFromApps, () => 'You must ensure all apps in this workspace are deletable'],
+        [isDeleteDisabledFromResources && isGoogleWorkspace(workspace), () => 'You must ensure all apps in this workspace are deletable'],
+        [isDeleteDisabledFromResources && isAzureWorkspace(workspace), () => 'This workspace cannot be deleted'],
         [_.toLower(deleteConfirmation) !== 'delete workspace', () => 'You must type the confirmation message'],
-        () => 'Delete Workspace')
+        () => '')
```

Comment:
This was causing a tooltip of "Delete Workspace" when the button was enabled, which doesn't really make sense. The tooltip should only be shown when the button is disabled.